docs/modules/devicetwin.md

```diff
@@ -0,0 +1,328 @@
+
+# DeviceTwin
+
+
+## Overview
+
+DeviceTwin is responsible for storing device status and syncing device status to the cloud. 
+It also provides query interfaces for applications.
+   
+   
+## Operations Performed By Device Twin Controller
+ 
+ The following are the functions performed by device twin controller :-
+  
+  - Sync Sql Lite
```

Comment:
can we rephrase this to "Sync metadata to/from db ( Sqlite )" . in the code it might be with this method name. but in the docs we can make it more clear.